app/assets/javascripts/discourse/app/lib/plugin-api.js

```diff
@@ -330,6 +330,9 @@ class PluginApi {
   /**
    * addPosterIcon(callback)
    *
+   * This function is an alias of addPosterIcons, which the latter has the ability
+   * to add multiple icons at once.
+   *
```

Comment:
Minor I'll suggesting removing the description below since it is already present in `addPosterIcons`

Reply:
I kind of thought about it... But I thought it'd be nice if the user can just see the usage directly from the function they are using. But alright. Since I am on the fence and you have an opinion, I'll do that.